scripts/typings/test_typings.js

```diff
@@ -0,0 +1,78 @@
+var spawnSync = require('child_process').spawnSync;
```

Comment:
Would be great to have an overview comment here about what this test is doing and why we need it.